### ISSUE TITLE
TNx as TNn

### ISSUE THREAD

**Issue by chunhsusu** [AusClimateService/indices#8]
We (Emma) have spotted a curious case where running,

```
# Example: TNx: Maximum daily minimum temperature
input_files=/g/data/xv83/agcd-csiro/tmin/daily/tmin_AGCD-CSIRO_r005_20200101-20201231_daily.nc
/g/data/xv83/dbi599/miniconda3/envs/icclim/bin/python /g/data/tp28/dev/chs548/indices/run_icclim.py --slice_mode month --verbose --input_files ${input_files} --variable tmin TNx icclim.out.nc
```

Produces a netcdf file that contains,

```
  variables:
    float TNx(time,lat,lon) ;
      TNx:_FillValue = NaNf ;
      TNx:standard_name = "minimum_air_temperature" ;
      TNx:long_name = "Minimum of daily minimum air temperature for each month." ;
      string TNx:units = "°C" ;
```

It should be MAXIMUM of daily minimum air temperature. We also found that the resulting values "TNx" are equivalent to "TNn".

My test case here: gadi: /scratch/public/chs548/test/

**Comment by ngben** [comment]
This doesn't seem to happen in previous version of icclim (5.4.0) on conda/analysis3-22.07. I think the bug snuck in in the 6.0.0 update.

**Comment by DamienIrving** [comment]
Well spotted! I've forked a copy of the icclim source code into the AusClimateService organisation and I've submitted a PR to the icclim team with a fix for this bug: https://github.com/cerfacs-globc/icclim/pull/243

While we wait for them to merge the bux fig and issue a new software release, would it be useful if I built a new conda environment (e.g. like `/g/data/xv83/dbi599/miniconda3/envs/icclim`) using my fork with the fixed bug?
